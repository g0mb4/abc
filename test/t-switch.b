main() {
    extrn putchar;
    auto i 0;

    while (i < 4) {
        switch i {
        case 3:
            putchar('3*n');
        case 2:
            putchar('2*n');
        case 1:
            putchar('1*n');
        }
        i++;
    }

    i = 0;
    while (i < 4) {
        switch i {
        case 0:
        case 2:
            putchar('e*n');
            goto end;
        case 1:
        case 3:
            putchar('o*n');
        }
end:
        i++;
    }
}
