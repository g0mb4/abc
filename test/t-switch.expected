1
2
1
3
2
1
e
o
e
o
